pom.xml

```diff
@@ -378,6 +378,8 @@
 
                         <!-- Exclude config and scripts -->
                         <exclude>**/dev-utils/*</exclude>
+                        <exclude>dev-support/artifacts/**</exclude>
```

Comment:
Also we should exclude this and   `dev-support/work/**` folders from `apache-rat-plugin` 
And maybe move those exclusions from a profile to apply globally but that could be handled as a separate task as it is not related to the scope of this change.

Reply:
Yes, that's not not strictly in scope, but makes testing the rat settings easier.